include/bitcoin/system/math/ec_scalar.hpp

```diff
@@ -61,9 +61,9 @@ class BC_API ec_scalar
     friend ec_scalar operator-(ec_scalar left, const ec_scalar& right);
     friend ec_scalar operator*(ec_scalar left, const ec_scalar& right);
 
-    using ec_secret_uniq = std::unique_ptr<ec_secret>;
+    using ec_secret_uptr = std::unique_ptr<ec_secret>;
```

Comment:
Given that `using` and `typedef` are functionally equivalent in aliasing, I don’t have a preference other than consistency. We have avoided `using` (except in one instance where there is no alternative) in class definitions (and headers generally, as free aliasing in headers leads to insidious namespace conflicts). We avoid `typedef` in implementation, preferring `using` in that context. We could change that at some point, but all things being equal consistency is the goal.

Reply:
I see no need for an alias here as it is never used except to declare the field.

Reply:
Ad-hoc acronyms are also avoided (along with abbreviations). We consistently use `ptr` for smart pointers where the distinction is important. As this is the only non-shared smart pointer in the codebase we could standardize on `uptr`, though I’d like to avoid proliferation of type identifiers in variable naming (see the Hungarian mess). So I’d prob just opt for `ptr` here. But for reasons previously mentioned, I think this would be better implemented as a shared pointer.